src/components/display.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../App.css';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types'; // eslint-disable-line import/no-extraneous-dependencies
+import { removeBook } from '../redux/books/books';
+
+const Display = ({ title, author, id }) => {
+  const dispatch = useDispatch();
+  const removeBookFromStore = (e) => {
+    e.preventDefault();
```

Comment:
- [ ] OPTIONAL - Consider changing this component name from Display to `Book` its more descriptive.